can_tools/scrapers/official/MO/mo_state.py

```diff
@@ -0,0 +1,233 @@
+    ).write(data.content)
+
+
+class MissouriState(MissouriBase):
+    """
+    Fetch details about overall Missouri state testing by county
+    """
+
+    headless = True
+    url = MissouriBase.source.format(tab="MetricsbyTestDatebyCounty")
+
+    # TODO0: ZW: The Data tab below is better... I think. Remove this once confirmed which data to use
+    #    # Download Crosstab from Tableau
+    #    tableau_click_selector_list: List[str] = [
+    #        "@data-tb-test-id = 'DownloadCrosstab-Button'",
+    #        "@data-tb-test-id = 'crosstab-options-dialog-radio-csv-Label'",
+    #        "@data-tb-test-id = 'export-crosstab-export-Button'"
+    #    ]
+
+    # Download Data from Tableau
+    tableau_click_selector_list: List[str] = [
+        "@data-tb-test-id = 'DownloadData-Button'",
+        _special_tableau_handler,
+    ]
+
+    async def _pre_click(self, page: pyppeteer.page.Page):
+        # TODO0: ZW: This page is slow to load. Adding wait.
+        #  Should probably find a better selector to use to determine that the page has loaded
+        import asyncio
+
+        await asyncio.sleep(5)
+        await page.waitForSelector("#main-content")
+
+    def _clean_cols(self, df: pd.DataFrame, bads: list, str_cols: list) -> pd.DataFrame:
+        for col in list(df):
+            for bad in bads:
+                df[col] = df[col].astype(str).str.replace(bad, "")
+            if col not in str_cols:
+                df[col] = pd.to_numeric(df[col])
+
+        return df
+
+    def normalize(self, df: pd.DataFrame) -> pd.DataFrame:
+        bads = [r",", r"%", "nan"]
+        str_cols = ["County", "FileNumber", "ProviderName"]
+        df = self._clean_cols(df, bads, str_cols)
+        df["location_name"] = df["County"].str.title()
+
+        # Rename appropriate columns
+        crename = {
+            "Bed Census": CMU(
+                category="hospital_beds_in_use", measurement="current", unit="beds"
+            ),
+            "Available": CMU(
+                category="hospital_beds_available", measurement="current", unit="beds"
+            ),
+            "Total Staffed Bed Capacity": CMU(
+                category="hospital_beds_capacity", measurement="current", unit="beds"
+            ),
+        }
+
+        # Drop grand total and melt
+        out = (
+            df.query("location_name != 'Grand Total'")
+            .melt(id_vars=["location_name"], value_vars=crename.keys())
+            .dropna()
+        )
+        out["value"] = pd.to_numeric(out["value"])
+        out = out.groupby(["location_name", "variable"]).sum().reset_index()
+
+        # Extract category information and add other context
+        out = self.extract_CMU(out, crename)
+        out["dt"] = self._retrieve_dt("US/Eastern")
+        out["vintage"] = self._retrieve_vintage()
+        # self.clean_desoto(out)
+        return out.loc[:, self.out_cols]
+
+
+# class MissouriICUUsage(MissouriHospitalUsage):
```

Comment:
What is the status of these commented out scrapers?

Reply:
They are copy/paste of Florida. They are just renamed so there is no naming conflicts. This page is just a calling point for the puppet changes.